### DISCUSSION TITLE
Static Calyx 2023 Summer Lab Notebook - Pai

### DISCUSSION THREAD

**Discussion by paili0628** [cucapra/futil#1563] [General]
Started this notebook to jot down progress. Will post updates each Monday.
For this week, I have been working on selective promotion and demotion. Also implementing Polybench test cases in Affine.

**Comment by rachitnigam** [comment]
Awesome, thanks for starting this!

**Comment by paili0628** [comment]
Started issue #1570 about how to apply egg to our compiler.
